Source/WebCore/platform/graphics/Path.cpp

```diff
@@ -200,6 +200,17 @@ void Path::addRoundedRect(const LayoutRoundedRect& rect)
     addRoundedRect(FloatRoundedRect(rect));
 }
 
+void Path::addContinuousRoundedRect(const FloatRect& rect, const float cornerRadius)
+{
+    if (rect.isEmpty())
+        return;
+
+    if (isEmpty())
+        m_data = PathSegment(PathContinuousRoundedRect { rect, cornerRadius, cornerRadius });
+    else
+        ensureProtectedImpl()->add(PathContinuousRoundedRect { rect, cornerRadius, cornerRadius });
```

Comment:
maybe, since you're creating the rounded rect instance anyway?
```suggestion
    PathContinuousRoundedRect roundedRect { rect, cornerRadius, cornerRadius };
    if (isEmpty())
        m_data = PathSegment(WTFMove(roundedRect));
    else
        ensureProtectedImpl()->add(WTFMove(roundedRect));
```